### ISSUE TITLE
Why not use bower as the asset source?

### ISSUE THREAD

**Issue by 13nightevil** [rmrevin/yii2-fontawesome#10]
I think it is better to use bower instead of clone the source. It's cleaner and lighter. When using bower, as the source of the assets, users don't need to download unnecessary assets (ex: src).
```
"require": {   
    "bower-asset/fontawesome": ">=4.3.0"
  },
```
and 
```php
public $sourcePath = '@bower/fontawesome';
```

**Comment by rmrevin** [comment]
We do not like the system to work with "bower" package through the "composer". But if there is such a need, we can realize.

**Comment by rmrevin** [comment]
Fixed in version 2.10.0
